molo/commenting/models.py

```diff
@@ -19,3 +21,9 @@ class MPTTMeta:
     class Meta:
         app_label = 'commenting'
         ordering = ['-submit_date', 'tree_id', 'lft']
+        
+@receiver(comment_was_flagged, sender=MoloComment)
+def remove_comment_if_flag_limit(sender, comment, flag, created, **kwargs):
+    if (comment.flags.count() >= 3):
```

Comment:
@KaitCrawford I meant "for now" - we'll have to introduce something similar to "Site Preferences" in mama, but that's a standalone feature that we can tackle later - its general function is to move settings that were typically found in the settings file into the admin.